metadata-ingestion/src/datahub/emitter/mce_builder.py

```diff
@@ -183,9 +183,17 @@ def make_term_urn(term: str) -> str:
 
 
 def make_data_flow_urn(
```

Comment:
Can we use typed DataFlowUrn instead of string? -> https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/utilities/urns/data_flow_urn.py

Reply:
Done